### DISCUSSION TITLE
How to close zsh start text?

### DISCUSSION THREAD

**Discussion by lmmarisej** [ohmyzsh/ohmyzsh#10987] [Help] [answered]
<img width="567" alt="image" src="https://user-images.githubusercontent.com/90127719/173192246-2f858095-d5e1-416a-ab05-693f236450ee.png">

when start or source etc. 

It's a bad experience for me every time I started.

**Comment by rwmitchell** [comment]
I don't understand what failed from your screenshot.  Could you be a little more specific?

**Reply by lmmarisej** [reply]
It's not a failure. I just want to turn off this prompt text every time I start zsh.

**Comment by rwmitchell** [comment, marked as answer]
You’ve added something to .zshrc that is doing this. It isn’t something OMZ does by default

**Reply by lmmarisej** [reply]
Oh, I get it. Thank u so much.

**Comment by lmmarisej** [comment]
The reason is that I installed fortune on brew and `fortune` was automatically added to the `~/.zshrc` file. Ummm... that was awkward.
